### DISCUSSION TITLE
[HTTP]: 500 on PnP - Search Results

### DISCUSSION THREAD

**Discussion by ascheuch** [microsoft-search/pnp-modern-search#3109] [Q&A] [answered]
****Version** used**
4.8 & 4.9 on different tenants

**Describe the bug**
"Since Friday I've been getting an error in Search Result web part: [HTTP]:500 - [CorrelationId]:a9cbc2a0-2035-6000-caee-9b5a2d3da46c [Version]:16.0.0.23821". The same problem on different pages, sites and users with the webpart in our tenant  and in my test tenant.

**To Reproduce**
1. Create a new page in Sharepoint
2. Place a PnP - Search Results
3. Select Sharepoint a datasource
4. Enter setting A, B, C
5. Set in the Query Template a path to a library

**Expected behavior**
The results as before

**Screenshots**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/89776216/92ccc993-feb8-4965-978e-ce156187b721)

**Desktop (please complete the following information):**
 - Browser Edge, Chrome and Firefox

**Additional context**
sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70 
        
        
        POST https://[ourTenant].sharepoint.com/sites/[ourSite]/_api/search/postquery 500
e.fetch @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
e._fetchCoreImpl @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
e.fetchCore @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
e._fetchWithInstrumentation @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
e.fetchCore @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
e._fetch @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
e.post @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
m @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
e.search @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
(anonym) @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
(anonym) @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
o @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
Promise.then (asynchron)
c @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
(anonym) @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
A @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
t.getData @ chunk.pnp-modern-search-sharepoint-search-datasource_036e4aa91a8adf843e72.js:1
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
U @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t.getDataFromDataSource @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
U @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t.componentDidMount @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
ss @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
bc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
t.unstable_runWithPriority @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Gi @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
hc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
ac @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Zs @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Bc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
(anonym) @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
rc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Wc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
t.render @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
t.renderCompleted @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
ys @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t.render @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t._renderWithAccessibleTitle @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
t._onInViewport @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._internalFirstTimeRender @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
e._renderWebPart @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
e.loadWebPart @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
ms @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Lc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ac @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
hc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
uc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
enqueueSetState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
h.setState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:3
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
t.getLoadOperations @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.onLoad @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.initialize @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.initialize @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
setTimeout (asynchron)
t.reinitialize @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.shouldComponentUpdate @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.shouldComponentUpdate @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
br @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Go @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ks @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
wc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Cc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
xc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
hc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
uc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
enqueueSetState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
h.setState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:3
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
a @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
notifyListeners @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
m @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
confirmTransitionTo @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
G @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
setTimeout (asynchron)
t._onIntercept @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._propagateLinkEvent @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._interceptAnchorClick @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._interceptClick @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._handleHeroItemClick @ sp-hero-webpart-bundle_de-de_bfb5523748f796739627.js:1
t._onItemClick @ sp-hero-webpart-bundle_de-de_bfb5523748f796739627.js:1
r._onClickItem @ sp-hero-webpart-bundle_de-de_bfb5523748f796739627.js:1
d.onClick @ sp-hero-layout_none_f8c11bfcd9f95a5704a5.js:1
Ve @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Qe @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
wa @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ea @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ue @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ta @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Zt @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Xt @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Pe @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Yt @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
modern-search-results-web-part_343763bbe6d907959f1e.js:2  Uncaught (in promise) Error: [HTTP]:500 -  [CorrelationId]:7bcdc2a0-00b9-6000-cd7b-b3fece47e5ea [Version]:16.0.0.23821
    at e.<anonymous> (modern-search-results-web-part_343763bbe6d907959f1e.js:2:1378540)
    at modern-search-results-web-part_343763bbe6d907959f1e.js:2:1376079
    at Object.next (modern-search-results-web-part_343763bbe6d907959f1e.js:2:1376184)
    at o (modern-search-results-web-part_343763bbe6d907959f1e.js:2:1374925)
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
U @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t.componentDidMount @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
ss @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
bc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
t.unstable_runWithPriority @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Gi @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
hc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
ac @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Zs @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Bc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
(anonym) @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
rc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
Wc @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
t.render @ react-dom-16-bundle_none_b1ebc9c6c87762929de1.js:2
t.renderCompleted @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
o @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
Promise.then (asynchron)
c @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
(anonym) @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
ys @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t.render @ modern-search-results-web-part_343763bbe6d907959f1e.js:2
t._renderWithAccessibleTitle @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
t._onInViewport @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._internalFirstTimeRender @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
e._renderWebPart @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
e.loadWebPart @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
ms @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Lc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ac @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
hc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
uc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
enqueueSetState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
h.setState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:3
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
Promise.then (asynchron)
t.getLoadOperations @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.onLoad @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.initialize @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.initialize @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
setTimeout (asynchron)
t.reinitialize @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.shouldComponentUpdate @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t.shouldComponentUpdate @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
br @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Go @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ks @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
wc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Cc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
xc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
hc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
qi @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
uc @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
enqueueSetState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
h.setState @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:3
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
a @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
notifyListeners @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
m @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
confirmTransitionTo @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
G @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
setTimeout (asynchron)
t._onIntercept @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._propagateLinkEvent @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._interceptAnchorClick @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._interceptClick @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:70
t._handleHeroItemClick @ sp-hero-webpart-bundle_de-de_bfb5523748f796739627.js:1
t._onItemClick @ sp-hero-webpart-bundle_de-de_bfb5523748f796739627.js:1
r._onClickItem @ sp-hero-webpart-bundle_de-de_bfb5523748f796739627.js:1
d.onClick @ sp-hero-layout_none_f8c11bfcd9f95a5704a5.js:1
Ve @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Qe @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
wa @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ea @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ue @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
(anonym) @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ta @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Zt @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Xt @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
IKDZ.t.unstable_runWithPriority @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Ki @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Pe @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69
Yt @ sp-pages-assembly_de-de_6d839d3a938e60f2594dadd800e8fa60.js:69

**Comment by wobba** [comment]
API issues are outside the scope of the web parts themselves and is a support issue to have with Microsoft.

**Comment by ascheuch** [comment]
OK thank you. I'll contact Microsoft support

**Reply by wobba** [reply, marked as answer]
It's unfortunate with random errors. You can try to turn off query rules if enables to see if this fix it. If you have a re-producable REST query you can also post at https://github.com/SharePoint/sp-dev-docs/issues which is triaged by Microsoft.
